docs/source/contrib/metrics.rst

```diff
@@ -8,4 +8,21 @@ Contribution module of metrics
 .. automodule:: ignite.contrib.metrics
    :members:
    :undoc-members:
-   :imported-members:
\ No newline at end of file
+   :imported-members:
+
+
+Regression metrics
+------------------
+
+Module `ignite.contrib.metrics.regression` provides implementations of
+metrics useful for the regression task. Definitions of metrics are based on `Botchkarev 2018`_, page 30 "Appendix 2. 
```

Comment:
```suggestion
metrics useful for the regression task. Definitions of metrics are based on `Botchkarev 2018`_, page 30 "Appendix 2". 
```

Reply:
Actually the full sentence is "Appendix 2. Metrics mathematical definitions".

Reply:
Maybe I should remove the line-break ?